tests/atest/rules/misc/keyword_after_return/expected_output.txt

```diff
@@ -1,2 +1,10 @@
 test.robot:20:5 [W] 0901 [Return] is not defined at the end of keyword. Note that [Return] does not quit from keyword but only set variables to be returned
-keyword_after_return_from.robot:21:5 [W] 0901 Keyword call after 'Return From Keyword'
\ No newline at end of file
+keyword_after_return_from.robot:21:5 [W] 0901 Keyword call after 'Return From Keyword'
+test.robot:57:9 [W] 0901 RETURN is not defined at the end of keyword
```

Comment:
Note that RETURN can be returned whenever, it's like ``return`` in Python:
```
IF    $condition
    RETURN
Other Keyword If Not Condition
```

The reason behind this rule is that previously RF only had ``[Return]`` and ``Return From Keyword`` (which worked as current ``RETURN``). ``[Return]`` didn't return from the keyword which was common misconception - it only set the variables which will be returned at the end of keyword. That's why I added this rule so the people will see that the whole code will always execute even if they put ``[Return]`` in the middle.

Reply:
Well, I think I haven't changed the logic of this rule, so not sure why it happened. I will double-check.

Reply:
Ok, I know what happened. I changed the file names.
Old `expected_output.txt` became new `expected_output_pre_rf5.txt` (because it is set to `target_version="<5"`) and the old `expected_output_rf5.txt` is now new `expected_output.txt` (because it should run with any other version of RF - `target_version=">=5"`, so previously the name was confusing).

Everything is fine here.